reana_client/cli/files.py

```diff
@@ -66,11 +66,12 @@ def files_group(ctx):
     default=None,
     help="Get URLs of output files.",
 )
+@click.argument("filename", metavar="SOURCE", nargs=1, required=False)
```

Comment:
Minor: for consistency this could be called `filenames` as in `download` and `upload` commands

Reply:
In `download` and `upload` you can pass multiple files, whereas for `ls` you can only pass a single expression for simplicity. Check `nargs` value that differs in the different cases.

Maybe it's something to be covered in the future, support use cases like `reana-client ls -w workflow 'data/*' 'results/*'`?